matplotlibrc.template

```diff
@@ -331,7 +331,11 @@
 #mathtext.tt:  monospace
 #mathtext.fallback_to_cm: True  # When True, use symbols from the Computer Modern
                                 # fonts when a symbol can not be found in one of
-                                # the custom math fonts.
+                                # the custom math fonts. Deprecated. 
+#mathtext.fallback: cm  # Select fallback font from ['cm' (Computer Modern), 'stix'
```

Comment:
@anntzer Any clue why this is apparently failing? The test passes for me on local

Reply:
This is because rcparams deprecation (fallback_to_cm) needs to be done by adding the rc to matplotlib._deprecated_remain_as_none, and (despite its name...) also removing it from the template.
See also https://github.com/matplotlib/matplotlib/pull/16528.